src/ai/backend/manager/api/session.py

```diff
@@ -259,7 +259,6 @@ def check_and_return(self, value: Any) -> object:
         t.Key("startup_command", default=None): t.Null | t.String,
         t.Key("bootstrap_script", default=None): t.Null | t.String,
         t.Key("owner_access_key", default=None): t.Null | t.String,
-        tx.AliasedKey(["scaling_group", "scalingGroup"], default=None): t.Null | t.String,
```

Comment:
We already check `"scaling_group`" here 

https://github.com/lablup/backend.ai/blob/9643a1471d2ea36da244f718cb241e2b196c5297/src/ai/backend/manager/api/session.py#L216